### DISCUSSION TITLE
Is there a way to add a description field to JsonSchema

### DISCUSSION THREAD

**Discussion by tobiasfoerg** [open-circle/valibot#871] [Q&A] [answered]
I would like to use the JsonSchema with Cloudflare worker AI. 

Tools on worker AI allow to specify a JsonSchema to be filled by the model. But requires a description on each field. Is there any way to add a description field?

**Comment by fabian-hiller** [comment, marked as answer]
Yes: https://valibot.dev/api/description/

**Reply by tobiasfoerg** [reply]
A nice. I somehow missed that. Thanks!
